src/ws/inactive.issue

```diff
@@ -0,0 +1,5 @@
+To activate your admin console run the following commands
```

Comment:
In this state it is not clear that this is a web/http thing. Also, could we make it one line?

So maybe: `To activate the web admin console, run: systemctl enable --now cockpit.socket`